src/UI/Art/SpriteArtist/MapSelectMenuArtist.java

```diff
@@ -108,7 +116,7 @@ private static void drawMapSelectMenu(Graphics g, MapSelectController gameSetup)
     int maxMiniMapHeight = drawableHeight - MapInfoPaneDrawHeight;
 
     // Find the center of the minimap display.
-    int miniMapCenterX = nameSectionDrawWidth + (maxMiniMapWidth / 2);
+    int miniMapCenterX = nameSectionDrawWidth + ((maxMiniMapWidth+1) / 2);
```

Comment:
Why +1? Is half the map width not the center of the map frame for some reason?

Reply:
I think it had a chance (with odd widths?) of overlapping the border between the two sections at certain sizes.
So I just said "no".

Reply:
Strange. Makes me think we must be off on some of the math somewhere, but maybe not worth hunting down.